cmd/juju/charmhub/info.go

```diff
@@ -26,7 +26,16 @@ The charm can be specified by name or by path.
 Channels displayed are supported by any base.
 To see channels supported for only a specific base, use the --base flag.
 --base can be specified using the OS name and the version of the OS, 
-separated by @. For example, --base ubuntu@22.04.
+separated by @.
+For example: --base ubuntu@22.04
+
+Use --revision to display information about a specific revision of the charm,
+which cannot be used together with --arch, --base, --channel or --series.
+For example: --revision 42
+
+Use --track to display information about a specific track of the charm,
```

Comment:
We need some extra text about how track is equivalent to channel of `<track>/stable` etc

Reply:
Some difference here is that, for example, `juju info postgresql` 

> with --channel stable
```
channels: |
  14/stable:     468  2024-09-11  (468)  28MB  amd64  ubuntu@22.04
  14/candidate:  ↑
  14/beta:       ↑
  14/edge:       ↑
```
> with --track 14
```
channels: |
  14/stable:     468  2024-09-11  (468)  28MB  amd64  ubuntu@22.04
  14/candidate:  545  2025-01-19  (545)  32MB  amd64  ubuntu@22.04
  14/beta:       545  2025-01-19  (545)  32MB  amd64  ubuntu@22.04
  14/edge:       551  2025-01-23  (551)  26MB  amd64  ubuntu@22.04
```
And we also support
> with --track 16
```
channels: |
  16/stable:     –
  16/candidate:  –
  16/beta:       –
  16/edge:       527  2024-11-27  (527)  32MB  arm64  ubuntu@24.04
```